invenio_drafts_resources/services/records/service.py

```diff
@@ -107,19 +107,23 @@ def search_drafts(
         # Prepare and execute the search
         params = params or {}
 
+        # `has_draft` systemfield is not defined here. This is not ideal
```

Comment:
why would we try to avoid overridding the method?

Reply:
i have no insights on this comment, i just moved the comment along with the code piece (see L117 on the left)